src/os/bluestore/BlueStore.h

```diff
@@ -272,6 +272,8 @@ class BlueStore : public ObjectStore,
   struct BufferSpace;
   struct Collection;
   struct Onode;
+  class Scanner;
+  class Estimator;
```

Comment:
@aclamk Does it make sense Estimator is not defined, but we use it here? better to let each commit pass the build.